calcom/cal.com
# feat: Cal.com Time Capsule

## What does this PR do?
Osshack Challenge Best Cal.com Time Capsule
Introduces a layout and data fetching to show user some stats from their cal.com account in the last year.



## Type of change



- [ x] New feature (non-breaking change which adds functionality)

## How should this be tested?
It uses the existing trpc api calls to get and load data. Testing would entail testing utils in the time capsule folder (pages/time-capsule/utils.ts). All data is fetched at the top of the component.



- What are the minimal test data to have?
      Some canceled bookings as well as past meetings for a user are needed to see live data 
- What is expected (happy path) to have (input and output)?
    -Numbers should display onscreen and reflect user data in Prisma


## Mandatory Tasks


## Checklist



- My code doesn't follow the style guidelines of this project
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
